core/blockchain.go

```diff
@@ -3375,3 +3375,22 @@ func (bc *BlockChain) SubscribeLogsEvent(ch chan<- []*types.Log) event.Subscript
 func (bc *BlockChain) SubscribeBlockProcessingEvent(ch chan<- bool) event.Subscription {
 	return bc.scope.Track(bc.blockProcFeed.Subscribe(ch))
 }
+
+// CheckContextAndOrderRange checks to make sure the range of a context or order is valid
+func (bc *BlockChain) CheckContextAndOrderRange(context int) error {
```

Comment:
Did you mean to also check the order of something, or should this be renamed to `CheckContextRange()`?

Reply:
This can be used to check the validity of both context, order, and path. Will change the name of the input variable to `number`.